src/index.html

```diff
@@ -378,11 +378,90 @@ <h2 class="page__section-title">
         </div>
       </section>
 
-      <section class="page__section" id="contacts">
+      <section class="page__section contact-us" id="contacts">
         <div class="container">
           <h2 class="page__section-title">
             Contact us
           </h2>
+
+          <div class="grid grid--deskAndTablet">
+            <div class="
+              grid__item
+              grid__item--tablet-1-3
+              grid__item--desktop-1-5
+            ">
+              <form action="#">
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  class="form-field contact-us__field"
+                >
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  class="form-field contact-us__field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                  form-field
+                  form-field--textarea
+                  contact-us__field
+                  contact-us__field--last
+                "></textarea>
+
+                <button class="button">
+                  Send
+                </button>
+              </form>
+            </div>
+            <div class="
+              grid__item
+              grid__item--tablet-4-5
+              grid__item--desktop-8-10
+            ">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+
+                <a
+                  href="+1 234 5555-55-55"
```

Comment:
In the contacts block: 404 error when clicking on a phone number (checklist p.5)